doc/manuals/user/known_limitations.md

```diff
@@ -29,29 +26,20 @@ Or, if you are sending a huge request, this one:
 carefully" text. Developers of the HTTP library in which Orion Context
 Broker is based seem to be funny guys :) :)
 
-If this 1 MB limit doesn't work for you, you can change it using the CLI option -payloadMaxSize.
-But, this is untested territory. At your own risk ...
+If this 1 MB limit doesn't work for you, you can change it using the [CLI option](../admin/cli.md) `-inReqPayloadMaxSize`.
+However, before doing this please have a look to [performance considerations](../admin/perf_tuning.md#payload-and-message-size-and-performance).
```

Comment:
look to => look at

Reply:
Fixed in e38ae720da8468a4cd2a35029c4f8c6a07dfa9c6